admin/scss/bootstrap/_variables.scss

```diff
@@ -135,7 +137,7 @@ $body-color:                 $gray-dark;
 // Set the number of columns and specify the width of the gutters.
 
 // $grid-columns:               12;
-// $grid-gutter-width:          1.875rem; // 30px
+$grid-gutter-width:          2.4615rem; // now 32px, was 1.875rem; ~ 30px
```

Comment:
Can we use formulas. Perhaps it would be better as expressing this as `32/13` assuming that there are 13px per rem in our default set-up.